lib/Qubit.class.php

```diff
@@ -52,9 +52,9 @@ public static function renderDate($value)
     $trim = '[-\/]0+';
 
     // Trim trailing zero day and month
-    $pattern = "/0*(\d+)(?:([-\/])$natural(?:([-\/])$natural
+    $pattern = "/0*(\\d+)(?:([-\\/])$natural(?:([-\\/])$natural
```

Comment:
I have doubts about this changes ... specially in the regular expressions ...

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.18/doc/rules/string_notation/escape_implicit_backslashes.rst

Reply:
I think it's okay in the end @djjuhasz ...

https://stackoverflow.com/a/23215075
https://www.php.net/manual/en/regexp.reference.escape.php

Render date method seems to be working okay and the model builder changes don't introduce changes in the actual models when they're built.